shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -342,3 +349,41 @@ async fn test_cassandra_request_throttling() {
 
     batch_statements::test(&connection);
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_events_keyspace() {
+    let _docker_compose =
+        DockerCompose::new("example-configs/cassandra-passthrough/docker-compose.yml");
+
+    let _shotover_manager =
+        ShotoverManager::from_topology_file("example-configs/cassandra-passthrough/topology.yaml");
+
+    let user = "cassandra";
+    let password = "cassandra";
+    let auth = StaticPasswordAuthenticatorProvider::new(&user, &password);
+    let config = NodeTcpConfigBuilder::new()
+        .with_contact_point("127.0.0.1:9042".into())
+        .with_authenticator_provider(Arc::new(auth))
+        .build()
+        .await
+        .unwrap();
+
+    let session = TcpSessionBuilder::new(RoundRobinLoadBalancingStrategy::new(), config)
+        .build()
+        .unwrap();
+
+    let mut event_recv = session.create_event_receiver();
+
+    sleep(Duration::from_secs(3)).await; // let the driver finish connecting to the cluster and registering for the events
+
+    let create_ks = "CREATE KEYSPACE IF NOT EXISTS test_events_ks WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };";
+
+    session.query(create_ks).await.unwrap();
+
+    let event = timeout(Duration::from_secs(10), event_recv.recv())
+        .await
+        .unwrap()
+        .unwrap();
+    assert!(matches!(event, ServerEvent::SchemaChange { .. }));
```

Comment:
Can we write like this to ensure that the server event comes from the creation of the keyspace we just did:
```rust
    assert_eq!(
        event,
        ServerEvent::SchemaChange(SchemaChange {
            change_type: SchemaChangeType::Created,
            target: SchemaChangeTarget::Keyspace,
            options: SchemaChangeOptions::Keyspace("test_events_ks".to_string())
        })
    );
 ```